tests/test_prompt_kit_hotkey_completion.py

```
@@ -0,0 +1,40 @@
+
+    def test_favorite_prompt_shortcuts_are_persisted_fail_closed(self) -> None:
+        source = POLISH.read_text(encoding="utf-8")
+        for marker in ("promptKit.promptShortcuts.v1", "prompt-kit-shortcuts/v1", "PROMPT_KIT_SHORTCUT_SEQUENCE_TIMEOUT_MS=1200", "function configurePromptShortcut(rawPromptId)", "if(!isFavoritePrompt(promptId))", "if(!persistPromptShortcutBindings(candidate))return false", "promptShortcutBindings=candidate", "function handleConfiguredPromptShortcutKey(e,key)", "showPromptDetail(promptId,null)"):
+            self.assertIn(marker, source)
+        self.assertLess(source.index("if(!persistPromptShortcutBindings(candidate))return false"), source.index("promptShortcutBindings=candidate"))
+
+    def test_configuration_ui_and_generated_parity_are_present(self) -> None:
+        source = POLISH.read_text(encoding="utf-8")
+        deployed = DEPLOYED.read_text(encoding="utf-8")
+        for marker in ("Favorite prompt shortcuts", "promptShortcutPromptId", "promptShortcutBindings", "Favorite a prompt, enter its ID", "Save favorite prompt keyboard shortcut"):
+            self.assertIn(marker, source); self.assertIn(marker, deployed)
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)